tests/lib/Microsoft.Fx.Portability.MetadataReader.Tests/Tests/OnlyNetStandardReference.il

```diff
@@ -0,0 +1,29 @@
+﻿///
```

Comment:
Cool - This is simpler to repro since you don't need to actually have the reference assemblies. I'll take your lead and switch the 2 mscorlib example to IL so it can be compiled without setting a Win7 VM up :)